src/Components/WelcomePage/WelcomePage.js

```diff
@@ -1,14 +1,16 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 import './WelcomePage.scss'
 import gsap from 'gsap';
 
 const WelcomePage = () => {
   const animate = () => {
     gsap.to(".logo", {duration: 3, fontSize: "30vw", ease: "power2", opacity: 100})
   }
+  useEffect(()=>{
```

Comment:
@nathanielmillard is this the hook implementation you were talking to me about? I'd love to hear about what this is doing and how we might be able to use more like it!

Reply:
Basically this is replacing ComponentDidMount(), since in a functional component we don't have access to life cycle methods, and refactoring this to a class component felt over engineered. But this allows for the function to fire after the elements of the component are rendered, avoiding the GSAP console error about the title componentent not existing.